src/bcd.c

```diff
@@ -581,6 +602,91 @@ bcd_kv_set(struct bcd_session *session, struct bcd_packet *packet)
 	    "malformed key-value pair");
 }
 
+static ssize_t
+bcd_arg_get(char **output, size_t n_output, bcd_error_t *error)
+{
+	struct bcd_arg *cursor;
+	size_t limit = n_output;
+	size_t i = 0;
+
+	if (bcd_arg_count == 0) {
+		return 0;
+	}
```

Comment:
Style: the rest of the code leaves out curly braces for conditionals if the code contained is one line.